modules/sb-install-service-catalog.adoc

```diff
@@ -7,6 +7,11 @@
 
 If you plan on using any of the services from the {asb-name} or {tsb-name}, you must install the service catalog by completing the following steps.
 
+The custom resources for the service catalog's API server and controller manager
+are created by default in {product-title}, but with a `managementState` of
```

Comment:
s/{product-title}, but with/{product-title} with
or 
s/{product-title}, but with/{product-title}, but initially have a

Reply:
Thanks for the suggestion @kalexand-rh, updated.